From ff240bc4c27cbc893a35bef27b9ac2da75973422 Mon Sep 17 00:00:00 2001
From: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>
Date: Mon, 27 Jun 2022 18:24:26 -0700
Subject: [PATCH] build: add linkage dependency against libva and libva-drm

If libva is installed in non-standard directory, you need
correct dependency to populate library location.

Signed-off-by: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>
---
 tools/cli/decvpp_tool/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/cli/decvpp_tool/CMakeLists.txt b/tools/cli/decvpp_tool/CMakeLists.txt
index 8c73b28..5e23812 100644
--- a/tools/cli/decvpp_tool/CMakeLists.txt
+++ b/tools/cli/decvpp_tool/CMakeLists.txt
@@ -73,8 +73,8 @@ if(UNIX)
       set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
       set(THREADS_PREFER_PTHREAD_FLAG TRUE)
       find_package(Threads REQUIRED)
-      target_link_libraries(${TARGET} ${PKG_LIBVA_LIBRARIES}
-                            ${PKG_LIBVA_DRM_LIBRARIES} ${PKG_THREAD_LIBRARIES})
+      target_link_libraries(${TARGET} PkgConfig::PKG_LIBVA
+                            PkgConfig::PKG_LIBVA_DRM Threads::Threads)
       target_include_directories(${TARGET} PUBLIC ${PKG_LIBVA_INCLUDE_DIRS})
     else()
       message(
-- 
1.8.3.1

